src/main/docs/guide/httpServer/jsonBinding.adoc

```diff
@@ -4,9 +4,30 @@
 
 The most common data interchange format nowadays is JSON.
 
-In fact, the defaults in the api:http.annotation.Controller[] annotation specify that the controllers in Micronaut framework consume and produce JSON by default.
+By default, the api:http.annotation.Controller[] annotation specifies that the controllers in Micronaut framework consume and produce JSON by default.
 
-To do so in a non-blocking manner, Micronaut framework builds on the https://github.com/FasterXML/jackson[Jackson] Asynchronous JSON parsing API and Netty, such that the reading of incoming JSON is done in a non-blocking manner.
+Since Micronaut Framework 4.0, users must choose how they want to serialize (Jackson Databind or Micronaut Serialization). Both approaches allow the usage of https://micronaut-projects.github.io/micronaut-serialization/snapshot/guide/index.html#jacksonAnnotations[Jackson Annotations].
```

Comment:
don't link to `snapshot` link to `latest`

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/9541/commits/0da3905083bfc76340d9d01310a5b1ceec2056b4